operator/config/overlays/cnsa/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Adds namespace to all resources.
+namespace: ibm-spectrum-scale-csi
+
+bases:
+- ../../manager/
+- ../../rbac/
+- ../../crd/
+- ./security_context_constraint.yaml
+
+patches:
+  - target:
+      kind: Deployment
+      labelSelector: "app.kubernetes.io/name=ibm-spectrum-scale-csi-operator"
+    patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: ibm-spectrum-scale-csi-operator
+      spec:
+        template:
+          spec:
+            containers:
+              - name: operator
+                env:
+                  # do not remove: this is required for CNSA; SHORTNAME_NODE_MAPPING="yes" for split hostNetwork node mapping.
+                  - name: SHORTNAME_NODE_MAPPING
+                    value: "yes"
+                  - name: CSI_SNAPSHOTTER_IMAGE
+                    value: cp.icr.io/cp/spectrum/scale/csi/csi-snapshotter@sha256:89e900a160a986a1a7a4eba7f5259e510398fa87ca9b8a729e7dec59e04c7709_TEST2
```

Comment:
Do we have any reason to have TEST2 at the end of image sha?

Reply:
Abhishek, this was added to test the remote kustomization in CNSA repo - will remove this in next commit